### PR TITLE
Re-implement quota API support in golang service broker.

### PR DESCRIPTION
**Pull request by bonzofenix**
Signed-off-by: Cross, Kevin <kevin.cross@sap.com>

### CODE REVIEW

File: .github/workflows/postgres.yaml [hunk 1]
```diff
@@ -9,9 +9,9 @@ jobs:
   postgres:
     strategy:
       matrix:
-        go: [1.16]
+        go: [1.15]
         java: [11]
-        postgres: [9.6, 12]
+        postgres: [12]
```
Comment (garethjevans) [inline .github/workflows/postgres.yaml:14]: we still need to support 9.6

File: .github/workflows/postgres.yaml [hunk 2]
```diff
@@ -9,9 +9,9 @@ jobs:
   postgres:
     strategy:
       matrix:
-        go: [1.16]
+        go: [1.15]
```
Comment (garethjevans) [inline .github/workflows/postgres.yaml:12]: we run with golang 1.16.7 in production

**Review by garethjevans — DISMISSED** [review]

File: .github/workflows/postgres.yaml (hunk 2 above)
Comment (asalan316) [reply .github/workflows/postgres.yaml:12]: Yes, I think, It was mistakenly changed becuase of act

File: .github/workflows/postgres.yaml (hunk 1 above)
Comment (asalan316) [reply .github/workflows/postgres.yaml:14]: Yes, because of debugging locally

File: .github/workflows/postgres.yaml (hunk 2 above)
Comment (KevinJCross) [reply .github/workflows/postgres.yaml:12]: Im trying 1.15 to see if the problem happens on 1.15.

File: .tool-versions [hunk 3]
```diff
@@ -0,0 +1,2 @@
+golang 1.16.8
+java adoptopenjdk-jre-11.0.12+7
```
Comment (asalan316) [inline .tool-versions:2]: I think, we need JDK (instead of JRE) to compile Java cod.

Comment (garethjevans) [reply .tool-versions:2]: @asalan316 correct - its waiting in here https://github.com/cloudfoundry/app-autoscaler/pull/720 for review, but I've got no idea why these commits are in here?

**Comment by garethjevans** [comment]
There are a pile of commits on this PR that aren't related to this? I'm not sure how they got there.  They should be removed and the PR squashed

File: src/autoscaler/go.mod [hunk 4]
```diff
@@ -1,6 +1,6 @@
 module autoscaler
 
-go 1.15
+go 1.16
```
Comment (garethjevans) [inline src/autoscaler/go.mod:3]: This is likely to cause issues until https://github.com/cloudfoundry/app-autoscaler-ci/blob/126386f314db9fe93827f429cbfb3667ac96640a/dockerfiles/autoscaler-test/Dockerfile#L43-L51 is updated (assume the deploy image needs the same)

**Review by aadeshmisra — APPROVED** [review]
LGTM

File: src/autoscaler/api/config/config.go [hunk 5]
```diff
@@ -68,15 +68,26 @@ type MetricsForwarderConfig struct {
...
+	Logging            helpers.LoggingConfig `yaml:"logging"`
+	BrokerServer       ServerConfig          `yaml:"broker_server"`
+	PublicApiServer    ServerConfig          `yaml:"public_api_server"`
+	DB                 DBConfig              `yaml:"db"`
+	BrokerUsername     string                `yaml:"broker_username"`
+	BrokerUsernameHash string                `yaml:"broker_username_hash"`
+	BrokerPassword     string                `yaml:"broker_password"`
+	BrokerPasswordHash string                `yaml:"broker_password_hash"`
+	APIClientId        string                `yaml:"api_client_id"`
+	//TODO: add yaml tag
```
Comment (garethjevans) [inline src/autoscaler/api/config/config.go:89]: we should fix this

**Pull request merged**